PRESUBMIT.py

```diff
@@ -1,6 +1,16 @@
-# Copyright 2018 The Chromium Authors. All rights reserved.
-# Use of this source code is governed by a BSD-style license that can be
-# found in the LICENSE file.
+# Copyright (c) 2018 The Khronos Group Inc.
```

Comment:
This is fine.  But we now usually put Google LLC for new files written by Google.  SPRV-Tools has mixed authorship.